docs/rfcs/014-born_digital_workflow/README.md

```diff
@@ -0,0 +1,176 @@
+# RFC 014: Born digital workflow
+
+**Last updated: 10 June 2019.**
+
+## Background
+
+As part of the move to our new [storage service](https://github.com/wellcometrust/platform/tree/master/docs/rfcs/002-archival_storage), we will be adopting [Archivematica](https://www.archivematica.org) to manage the preservation workflow for born-digital archives. This is analogous to our use of [Goobi](https://www.intranda.com/en/digiverso/goobi/goobi-overview/) for managing digitisation workflows.
+
+This document describes an initial workflow that will allow us to start using Archivematica as soon as possible. This means that, for now, we will continue to perform some parts of the workflow manually, outside of Archivematica.
+
+We will be actively contributing to the development of Archivematica, including ensuring that it better addresses our needs around appraisal and cataloguing. This will mean we are able to streamline the workflow over time, as the amount of born-digital material increases.
+
+## Proposed solution
+
+### Principles
+
+There are three key principles underlying this initial workflow:
+
+- Appraisal and selection will remain manual until the functionality in Archivematica improves
+- Cataloguing will be done in Calm and file-level description will not be part of the catalogue record
+- All processing steps within Archivematica will be fully automated and require no intervention
+
+### Overview
+
+![Workflow diagram](workflow.png)
+
+The workflow consists of the following high-level stages:
+
+1. Appraisal
+2. Cataloguing
+3. Transfer
+4. Processing
+5. Storage
+
+Each of these are described below.
+
+### Appraisal
+
+During the appraisal stage, an archivist will determine which files are in scope for the collections and perform a policy-led sensitivity check. We intend to move towards system-assisted appraisal, using future version of Archivematica, but for now this stage will build on existing manual processes.
+
+| Step                 | Description |
+|----------------------|-------------|
+| Disk level appraisal | High level appraisal to determine what storage media is in scope. |
+| File level appraisal | Selection of in scope files from selected storage media. |
+| Sensitivity check    | Policy-led review of selected files to determine whether they contain sensitive material. |
```

Comment:
We don't really do a proper sensitivity check at this stage. We will flag some likely issues such as 'contains patient records' or 'some of the correspondence is highly personal'. Sensitivity checking would happen as part of the cataloguing process.